### PR TITLE
add service account changes

### PR DESCRIPTION
**Pull request by ericgraf**
## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Review by MarcelMue — APPROVED** [review]

**Comment by MarcelMue** [comment]
Can you mark this ready for review? I might want to merge it some time tomorrow :)

**Review by stone-z — APPROVED** [review]
LGTM once tested :rocket:

File: files/manifests/k8s-api-server.yaml [hunk 1]
```diff
@@ -68,6 +68,9 @@ spec:
     - --requestheader-extra-headers-prefix=X-Remote-Extra-
     - --requestheader-group-headers=X-Remote-Group
     - --requestheader-username-headers=X-Remote-User
+    - --service-account-issuer=https://kubernetes.default.svc.cluster.local/
```
Comment (ericgraf) [inline files/manifests/k8s-api-server.yaml:71]: Need to still make this one not hard coded after we prove it works,

**Review by whites11 — APPROVED** [review]
LGTM

**Pull request merged**